src/NzbDrone.Core/Notifications/Notifiarr/NotifiarrSettings.cs

```diff
@@ -0,0 +1,28 @@
+using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Notifications.Notifiarr
+{
+    public class NotifiarrSettingsValidator : AbstractValidator<NotifiarrSettings>
+    {
+        public NotifiarrSettingsValidator()
+        {
+            RuleFor(c => c.APIKey).NotEmpty();
+        }
+    }
+
+    public class NotifiarrSettings : IProviderConfig
+    {
+        private static readonly NotifiarrSettingsValidator Validator = new NotifiarrSettingsValidator();
+
+        [FieldDefinition(0, Label = "API Key", Privacy = PrivacyLevel.ApiKey, HelpText = "Your API key from your profile", HelpLink = "https://notifiarr.com")]
+        public string APIKey { get; set; }
```

Comment:
`ApiKey` to maintain consistency with other naming

Reply:
I think Marcus wanted the _variable_ named ApiKey, not change the label ;)